seedbuilder/areas.ori

```diff
@@ -1665,53 +1675,71 @@ home: BashTree 521 326
 	conn: BashTreeDoorClosed
 		casual Open
 	conn: UpperGinsoRedirectArea
-		casual Bash
-		casual ChargeJump Dash DoubleJump
+		casual Bash WallJump
+		casual Bash Climb
+		casual Bash ChargeJump
+		casual Bash Grenade
 		casual ChargeJump DoubleJump Glide
+		standard ChargeJump Dash DoubleJump
```

Comment:
This path should actually be two different paths - a standard path for air dash, and an expert path for dash glide:
```
standard ChargeJump AirDash DoubleJump
expert ChargeJump Dash DoubleJump
```